### DISCUSSION TITLE
Mouse can't freely move to other monitor(s)

### DISCUSSION THREAD

**Discussion by ppenguin** [hyprwm/Hyprland#1283] [Q&A]
I think I didn't configure anything special regarding mouse focus, but I can't simply move my mouse across monitors, which however is very much expected standard behavour. Now my mouse pointer just stops at the monitor boundary.

In fact, when trying to drag a floating window over, it also doesn't appear, as if the desktop area is in fact not recognised as being contiguously connected, even though it appears to be:

```
> hyprctl monitors | grep Monitor -A1
Monitor DP-1 (ID 0):
        3840x2160@59.997002 at 0x0
--
Monitor DP-2 (ID 1):
        3840x2160@59.997002 at 3840x0
```

(I.e. the second monitor is directly to the right of the first one)

Do I have to enable this functionality somehow, or are there any ideas how to troubleshoot this?

(My keybinds to focus another (next/prev) monitor do work correctly, i.e. they focus another monitor and warp the mouse cursor there too. The *additional* behaviour I'm seeking is that monitor focus will follow the mouse as well if I move it to the other monitor.)

**Comment by ppenguin** [comment]
:man_facepalming: 
Oh, I found out that if using scaling, that I should take the position into account with the scaled resolution, in my case (`scale=1.5`) this means the second monitor position should be `2560x0` (`3840 / 1.5 = 2560`)

Sorry for the noise

**Reply by vaxerski** [reply]
reading the wiki would help

**Reply by ppenguin** [reply]
Yeah, I noticed the same, I was a bit confused by the wording so I didn't find it before. This would also be a good FAQ entry I suppose (at least as a link to the wiki).

One of the reasons I didn't even think about it is that it doesn't seem logical from a `wayland` point of view? I mean: the scaling will tell clients to "project" their content in a higher resolution, but the physical resolution of the viewport is still 1:1, so it would make more sense if the layout uses the physical resolution?

**Reply by vaxerski** [reply]
umm no not how scaling works.

**Reply by deeDubitGuy** [reply]
thanks, I stuggled with this for a couple of days and the wiki only made sense in the rear view mirror
